src/System.Net.WebSockets.Client/src/Resources/Strings.resx

```diff
@@ -121,4 +121,10 @@
   <data name="net_securityprotocolnotsupported" xml:space="preserve">
     <value>The requested security protocol is not supported.</value>
   </data>
+  <data name="net_WebSockets_UWPClientCertSupportRequiresWindows10InsiderPreviewBuild16215OrGreater" xml:space="preserve">
+    <value>Support for client certificates in UWP requires Windows 10 Insider Preview Build 16215 or greater.</value>
```

Comment:
Agreed. Although literally true, we should make this more generic. Something like, "... unsupported in Windows 10 version 1703 and earlier". As of yesterday, the insider preview build is version 16232, so there's quickly diminishing relevance of baking in the RS3 version.

---
In reply to: [126160835](https://github.com/dotnet/corefx/pull/21908#discussion_r126160835) [](ancestors = 126160835)